runelite-client/src/main/java/net/runelite/client/game/GameArea.java

```diff
@@ -0,0 +1,531 @@
+package net.runelite.client.game;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.coords.WorldArea;
+
+@Getter
+public enum GameArea
+{
+	// Bosses
+	BOSS_ABYSSAL_SIRE("Abyssal Sire", GameAreaType.BOSSES, 11851, 11850, 12363, 12362),
+	BOSS_CERBERUS("Cerberus", GameAreaType.BOSSES, 4883, 5140, 5395),
+	BOSS_COMMANDER_ZILYANA("Commander Zilyana", GameAreaType.BOSSES, new RegionArea(11602, new WorldArea(2889, 5258, 19, 18, 0))),
+	BOSS_CORP("Corporeal Beast", GameAreaType.BOSSES, 11842, 11844),
+	BOSS_DKS("Dagannoth Kings", GameAreaType.BOSSES, 11588, 11589),
+	BOSS_GENERAL_GRAARDOR("General Graardor", GameAreaType.BOSSES, new RegionArea(11347, new WorldArea(2864, 5351, 13, 19, 2))),
```

Comment:
Will the `WorldArea`s within this work in instanced versions of these regions? e.g. Ironman instance of Bandos here

Reply:
No, they are drawn for the non-instanced map. `DiscordGameEventType.fromPoint(WorldPoint)` expects to be given a point adjusted for instances. (eg. from `WorldPoint.fromLocalInstance(Client, LocalPoint)`)